packages/common/src/utils/Helpers/convertSigner.ts

```diff
@@ -0,0 +1,57 @@
+import { EthersSigner } from "../EthersSigner";
+import { SupportedSigner } from "../Types";
+import { WalletClient } from "viem";
+import { WalletClientSigner, SmartAccountSigner } from "@alchemy/aa-core";
```

Comment:
what's stopping up from not importing stuff from alchemy/aa-core?
cc @AmanRaj1608

Reply:
I don't follow

Reply:
![image](https://github.com/bcnmy/biconomy-client-sdk/assets/4627587/b8c02385-3a2a-45e1-8e7b-d1a10c65f7e7)
Our main BiconomySmartAccountV2 class extends it

Reply:
Hmm I was thinking to create a local copy of this so every time we are not forced to update and wait for alchemy to make some changes.